modules/appeals_api/spec/services/appeals_api/pdf_construction/generator_spec.rb

```diff
@@ -225,6 +225,28 @@
         end
       end
 
+      context 'pdf verification alternate signer' do
```

Comment:
I think using the [maxlength tests](https://github.com/department-of-veterans-affairs/vets-api/blob/673c4f96711821eda1117b476357d1cec0547feb/modules/appeals_api/spec/services/appeals_api/pdf_construction/generator_spec.rb#L93) as a basis is the way to go here, rather than adding new factories & data fixture files. Adjust the headers as you want before you the generator gets run and then compare it against the expected alt signer pdf you've already generated.

Reply:
maxlength test are disabled